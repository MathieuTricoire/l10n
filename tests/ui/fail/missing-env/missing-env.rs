l10n::init!();

fn main() {}
