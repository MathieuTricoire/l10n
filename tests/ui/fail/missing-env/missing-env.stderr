error: l10n path for environment "prod" is not set in the configuration
 --> tests/ui/fail/missing-env/missing-env.rs:1:1
  |
1 | l10n::init!();
  | ^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `l10n::init` (in Nightly builds, run with -Z macro-backtrace for more info)
